actions/app.go

```diff
@@ -38,8 +44,21 @@ func App() *buffalo.App {
 			app.Use(middleware.ParameterLogger)
 		}
 
-		app.GET("/", HomeHandler)
+		// middleware
+		app.Use(authenticateRequest)
+
+		authGroup := app.Group("/auth")
+		authGroup.Middleware.Skip(authenticateRequest, createUserHandler, loginHandler) // do not verify a token for these registration/login handlers
+		authGroup.POST("/user", createUserHandler)
+		authGroup.POST("/login", loginHandler)
+
+		app.GET("/user/{userID}", userDataHandler)
+		app.PUT("/user/monster", renameMonsterHandler)
```

Comment:
Better way to do this than taking in No and Name as params? Possibly send the monster's _id as the header?

Reply:
yeah we can send the monster's _id as a form parameter in the request body